examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
```

Comment:
I would like to have a burstable instance, but if online can't do this I will remove it.